app/tests/routes/wallet/dids/test_create_did.py

```diff
@@ -1,21 +1,94 @@
 from unittest.mock import AsyncMock, patch
 
 import pytest
-from aries_cloudcontroller import DIDCreate
 from aries_cloudcontroller.exceptions import (
     ApiException,
     BadRequestException,
     NotFoundException,
 )
 
+from app.models.wallet import DIDCreate
 from app.routes.wallet.dids import create_did
 
 
 @pytest.mark.anyio
 @pytest.mark.parametrize(
-    "request_body", [None, DIDCreate(method="key"), DIDCreate(method="sov")]
+    "request_body, create_body",
+    [
+        (None, None),
+        (
+            DIDCreate(method="key"),
+            DIDCreate(
+                method="key", options={"key_type": "ed25519"}, key_type="ed25519"
+            ),
+        ),
+        (
+            DIDCreate(method="sov"),
+            DIDCreate(
+                method="sov", options={"key_type": "ed25519"}, key_type="ed25519"
+            ),
+        ),
+        (
+            DIDCreate(method="did:peer:2"),
+            DIDCreate(
+                method="did:peer:2", options={"key_type": "ed25519"}, key_type="ed25519"
+            ),
+        ),
+        (
+            DIDCreate(method="did:peer:4"),
+            DIDCreate(
+                method="did:peer:4", options={"key_type": "ed25519"}, key_type="ed25519"
+            ),
+        ),
+        (
+            DIDCreate(method="key", key_type="bls12381g2"),
+            DIDCreate(
+                method="key", options={"key_type": "bls12381g2"}, key_type="bls12381g2"
+            ),
+        ),
+        (
+            DIDCreate(method="sov", key_type="bls12381g2"),
+            DIDCreate(
+                method="sov", options={"key_type": "bls12381g2"}, key_type="bls12381g2"
+            ),
+        ),
+        (
+            DIDCreate(method="did:peer:2", key_type="bls12381g2"),
+            DIDCreate(
+                method="did:peer:2",
+                options={"key_type": "bls12381g2"},
+                key_type="bls12381g2",
+            ),
+        ),
+        (
+            DIDCreate(method="did:peer:4", key_type="bls12381g2"),
+            DIDCreate(
+                method="did:peer:4",
+                options={"key_type": "bls12381g2"},
+                key_type="bls12381g2",
+            ),
+        ),
+        (
+            DIDCreate(method="web", did="did:web:1234"),
+            DIDCreate(
+                method="web",
+                options={"key_type": "ed25519", "did": "did:web:1234"},
+                key_type="ed25519",
+                did="did:web:1234",
+            ),
+        ),
+        (
+            DIDCreate(method="web", key_type="bls12381g2", did="did:web:1234"),
+            DIDCreate(
+                method="web",
+                options={"key_type": "bls12381g2", "did": "did:web:1234"},
+                key_type="bls12381g2",
+                did="did:web:1234",
+            ),
```

Comment:
Just to double check -- the first DIDCreate is the request we get in from the client, and the 2nd one is what we pass to the acapy agent -- through the acapy_wallet.create_did method.  

But, acapy only knows about the options field - not the extra key_type + did fields. They'll just be ignored.

So it'll be more consistent to ensure that the method being passed to acapy just has the method and options